features/support/helpers/shared-helpers.js

```diff
@@ -26,6 +26,11 @@ export const waitUntilTextInSelector = async (client, { selector, text }) => (
   })
 );
 
+export const waitAndSelect = async (client, selectSelector, waitSelector = selectSelector, ...waitArgs) => {
```

Comment:
@daniloprates please rename this function to `getVisibleElementsForSelector` and also add another helper function which will call this one and return just the counts of elements so that in your step-definition you don't have to use `addresses.value.length`.
This second helper could be named: `getVisibleElementsCountForSelector` (it would basically call `getVisibleElementsForSelector` and then return `value.length` of it's result...